crates/ruff/src/rules/ruff/mod.rs

```diff
@@ -196,4 +196,14 @@ mod tests {
         assert_messages!(snapshot, messages);
         Ok(())
     }
+
+    #[test]
+    fn ruff_static_key_dict_comprehension() -> Result<()> {
+        let diagnostics = test_path(
+            Path::new("ruff/RUF011.py"),
+            &settings::Settings::for_rules(vec![Rule::StaticKeyDictComprehension]),
+        )?;
+        assert_messages!(diagnostics);
+        Ok(())
+    }
```

Comment:
I think we can add this as a `#[test_case]` on the `rules` function above:
```rust
#[test_case(Rule::StaticKeyDictComprehension, Path::new("RUF011.py"))]
```

Reply:
Great catch, seems like the test for ruff_pairwise_over_zipped could use this treatment too, but that's out of scope for this PR